### PR TITLE
Set babel target to ie11, refactor and add polyfills

### PR DESCRIPTION
**Pull request by Delerme**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "ie": "11"
+        },
+        "useBuiltIns": "entry"
+      }
+    ]
+  ]
```
Comment (b1lly) [inline .babelrc:12]: Lets remove all the polyfilling for now.

Keep the merge change, though.

**Review by b1lly — CHANGES_REQUESTED** [review]
Lets remove all polyfilling, but keep the merge object change for now.

**Comment by b1lly** [comment]
Should I close this PR?

**Comment by Delerme** [comment]
Yup, I got it

**Pull request closed without merging**